StantUiIosLibrary/Classes/Cells/CellWithIconAndDescriptionCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CellWithIconAndDescriptionCell.swift
+//  Pods-StantUiIosLibraryDemo
+//
+//  Created by Alexandre on 01/07/19.
+//
+
+import UIKit
+
+public class CellWithIconAndDescriptionCell: UITableViewCell {
```

Comment:
On others components, I was trying to give names for the components according to redesign. 
What do you think about changing it to Intern Cell / Action?

Reply:
My bad, I was trying to figure out a name that would be good to be open, but I will change it. Does InternActionCell sounds good?